be1-go/hub/organizer/mod_test.go

```diff
@@ -460,12 +462,11 @@ type keypair struct {
 }
 
 var nolog = zerolog.New(io.Discard)
-var suite = crypto.Suite
+//var suite = crypto.Suite
```

Comment:
Does it still work if you remove the variable ? It seems to be used in generateKeyPair

Reply:
I moved it to mod.go